runtime/vm/classloadersearch.c

```diff
@@ -227,6 +229,9 @@ addZipToLoader(J9JavaVM * vm, const char * filename, J9ClassLoader * classLoader
 			if ((*env)->ExceptionCheck(env)) {
 				rc = CLS_ERROR_OUT_OF_MEMORY;
 			}
+			if 	((J2SE_VERSION(vm) >= J2SE_19) && (J2SE_SHAPE(vm) >= J2SE_SHAPE_B165)) {
```

Comment:
This code should not run if the above code errored.  Need an else or goto cleanup.